app/editor/src/features/admin/topics/TopicList.tsx

```diff
@@ -39,6 +39,24 @@ export const TopicList: React.FC = () => {
 
   const topicTypeOptions = getEnumStringOptions(TopicTypeName);
 
+  React.useEffect(() => {
+    // It looks like we could not use useRef in this case due to the
+    // handling needs to be dealt with here instead of in parent component
+    const main = document.querySelector('main') as HTMLElement;
```

Comment:
If I'm reading this correctly you are manually querying the DOM and then manually setting an inline style property the first time this component is created.  

All we want is the CSS to correctly be setup.

@jdtoombs can you help @CelloLiu with CSS and styled components.

Reply:
Updated a little bit, and please let me know if there is a better way for this :) Thanks